src/pages/DelegatoryValidator/StakeOperationDialog.tsx

```diff
@@ -92,7 +91,7 @@ export default function StakeOperationDialog({
     if (isAmountValid) {
       await submitStakeOperation(
         validator.owner_address,
-        Number(amount),
+        Number(Number(amount).toFixed(8)),
```

Comment:
can use BigInt()..?
https://github.com/aptos-labs/explorer/blob/main/src/api/index.ts#L35

Reply:
i don't think it worked: if we just BigInt(a number), if that number exceeds the bigint allowed limit, it will still throw errors
<img width="258" alt="Screenshot 2023-02-21 at 10 11 27 AM" src="https://user-images.githubusercontent.com/121921928/220425894-f49e55c4-7210-48d5-93c0-2b61bc632eb3.png">